src/zeit/content/article/edit/browser/social.py

```diff
@@ -27,6 +27,11 @@ def __init__(self, context, request):
             zeit.content.article.interfaces.IArticle).select(
                 'is_amp', 'is_instant_article')
 
+    def setUpWidgets(self, *args, **kw):
+        super(Social, self).setUpWidgets(*args, **kw)
+        if self.context.access != u'free':
```

Comment:
Da `free` bereits als default-Wert für `Access` im Quellcode verankert ist, prüfen wir hier und im Event-Handler ebenfalls auf `free` um nicht weitere Abhängigkeiten von festen Werten die eigentlich per XML ausgesteuert werden reinzubringen. Sauberer wäre in der XML Quelle für Access ein `restricted` Attribut einzuführen. Das war uns aber zu viel Overhead. Findest du das akzeptabel so @wosc? Oder sauberer per XML Quelle lösen? Oder zumindest Konstante einführen?